internal/autocert/manager.go

```diff
@@ -105,11 +106,15 @@ func (mgr *Manager) updateAutocert(cfg *config.Config) error {
 			cert, err = cm.CacheManagedCertificate(domain)
 		}
 		if err == nil && cert.NeedsRenewal(cm) {
+			expired := time.Now().After(cert.Leaf.NotAfter)
 			log.Info().Str("domain", domain).Msg("renewing certificate")
 			err = cm.RenewCert(context.Background(), domain, false)
-			if err != nil {
+			if err != nil && expired {
 				return fmt.Errorf("autocert: failed to renew client certificate: %w", err)
 			}
+			if !expired {
```

Comment:
Will this log even if cert successfully renews since you don't update expired after cm.RenewCert?

Reply:
Ah thanks, fair point 👍

Reply:
Fixed.